_misc/gather_i18n.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+from __future__ import unicode_literals
+
+import argparse
+import logging
+import click
```

Comment:
It's only a requirement for these development CLI tools. You can install it by hand, `pip install click` will do.